src/cache/client.py

```
@@ -0,0 +1,68 @@
+logger = logging.getLogger(__name__)
+
+# Initialize cache at module import time (synchronously)
+# This ensures ContextVars are properly initialized before any async contexts
+if settings.CACHE.ENABLED:
+    try:
+        cache.setup(  # pyright: ignore[reportUnknownMemberType]
+            settings_url=settings.CACHE.URL,
+            pickle_type=PicklerType.SQLALCHEMY,  # Use sqlalchemy for SQLAlchemy object serialization
+        )
+        logger.debug("Cache setup completed at module import")
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)